src/rollup/recommendations/recommend-content.js

```diff
@@ -55,7 +55,7 @@ export default function recommendContent (accountId, params, id, callback) {
   var seerId = readCookie('seerid');
 
   if (!seerId) {
-    throw new Error('Cannot find SEERID cookie');
+    callback([]);
```

Comment:
this is fine but do we need to ensure that callback is a function here and above ( i realize the above is already in production ). ever a chance of not having a callback function here?

Reply:
Should probably never happen, but I'll add a type check in case of user error.

Reply:
better safe than sorry :)